### DISCUSSION TITLE
Fail to compile

### DISCUSSION THREAD

**Discussion by chencc33** [lum-ai/odinson#380] [Q&A] [answered]
I have been trying to set up Odinson on different operating systems, mac and windows. However, it keeps giving me the same errors (Please see the attached picture). I have tried to use different versions of java, scala, however, it does not work. Do you happen to know how should I solve this problem?
![Screen Shot 2022-04-21 at 1 07 29 AM](https://user-images.githubusercontent.com/47489094/164409461-987b45b6-d192-4b24-8b66-75b51ae979f6.png)

**Comment by kwalcock** [comment, marked as answer]
Judging from the directory name that includes odinson-master, I would guess that you downloaded a zipped version and expanded it.  There is something in build.sbt:65 that expects information from git and won't work on the zipped version.  It needs to work from a clone that has the .git directory.  Can you do a "git clone" instead?

**Reply by chencc33** [reply]
Thanks! I tried git clone this time and it gives me different errors. It seems that some of the processor models no longer exist?
![Screen Shot 2022-04-21 at 10 54 56 AM](https://user-images.githubusercontent.com/47489094/164521476-f2e4ac97-713d-4e11-a01a-7c9699224466.png)

**Reply by kwalcock** [reply]
That's strange.  The models do exist, but not at that maven location.  There should be a line in build.sbt like the one below.  Perhaps we've lost it and everyone else has processors installed separately so that this isn't needed.  Can you add this line in a reasonable place and run sbt again?

// for processors-models
resolvers += ("Artifactory" at "http://artifactory.cs.arizona.edu:8081/artifactory/sbt-release").withAllowInsecureProtocol(true)

**Reply by chencc33** [reply]
Thank you so much! It works now!
